cpp/src/sqls/sqls_rtti_comp.h

```diff
@@ -28,6 +28,8 @@
 #include "utilities/type_dispatcher.hpp"
 #include "bitmask/legacy_bitmask.hpp"
 

```

Comment:
Why can't we just eliminate this file now?

Reply:
A lot of this is still used, yet not used. What i mean is that its still used for sort based group by, which is not exposed and therefore not used, and its used in gdf_filter, which I propose we get rid of because it is in functions.h but its not on the python side. Now, I could go get rid of all of that in this PR, but I did not want to make this PR too big. If you want, I can go ahead and purge all this

Reply:
I was not aware of gdf_filter. It is/will be superseded by cudf::copy_if (used by apply_boolean_mask and drop_nulls), which has null support.

Reply:
> NOTE, i have added #1099 into this PR so that, #1099 can be merged in and then this one can be merged in easily, since some of the functions being modified by #1099 I move into another file

@williamBlazing I'm not sure what you mean. #1099 is an issue, not a PR. Issues can't be merged. :)